python/paddle/fluid/dygraph/base.py

```diff
@@ -52,6 +62,7 @@ def __impl__(*args, **kwargs):
 
 
 no_grad = wrap_decorator(_no_grad_)
+dygraph_not_support = wrap_decorator(_dygraph_not_support_)
```

Comment:
Too long name. Users have to write something like `dygraph.dygraph_not_support`. Please simplify it to be `not_supported`.